src/test/util_sds_check/util_sds_check.h

```diff
@@ -0,0 +1,58 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright 2025, Hewlett Packard Enterprise Development LP */
+
+/*
+ * util_sds_check.h -- header for unit test
+ */
+
+#include <stdlib.h>
+#include <stdbool.h>
+#include "unittest.h"
+#include "shutdown_state.h"
+#include "set.h"
+
+int os_part_deep_common(struct pool_replica *rep, unsigned partidx, void *addr,
+	size_t len, int flush);
+/* This function does not take part in this unit test. */
+int
+os_part_deep_common(struct pool_replica *rep, unsigned partidx, void *addr,
+	size_t len, int flush)
+{
+	return 0;
+}
+
+#define DIRTY_SET 1
+#define DIRTY_CLEAR 0
+
+/* Invalid value for all test cases */
+#define INVALID_VALUE 0
+
+#define CHECKSUM 0
+#define UUID 1
+#define USC 1
+
+#define SDS_CUSTOM(USC, UUID, DIRTY_BIT, CHECKSUM) \
+{ \
+	.usc = USC, \
+	.uuid = UUID, \
+	.dirty = DIRTY_BIT, \
+	.reserved = {0}, \
+	.checksum = CHECKSUM \
+}
```

Comment:
```suggestion
#define SDS_CUSTOM(usc, uuid, dirty, checksum) \
{ \
	.usc = usc, \
	.uuid = uuid, \
	.dirty = dirty, \
	.reserved = {0}, \
	.checksum = checksum \
}
```

IMHO pre-processor could be seriously confused which value ought to be set in line 36. I am for sure. 😆 I recommend using different literals than the ones already defined e.g. using small letters.

Reply:
.